app/models/goal.py

```diff
@@ -4,3 +4,5 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
```

Comment:
Should we be able to create a goal with a NULL title? Consider adding `nullable=False` here.